src/utils/dateHelpers.test.ts

```diff
@@ -79,6 +80,32 @@ describe('diffMinutes', () => {
     });
 });
 
+describe('diffSeconds()', () => {
+    it('should return 45 seconds difference between two dates', () => {
+        // given
+        const date1 = new Date('2020-01-20T23:00:00');
+        const date2 = new Date('2020-01-20T23:00:45');
+
+        // when
+        const result = diffSeconds(date1, date2);
+
+        // then
+        expect(result).toEqual(45);
+    });
+
+    it('should return with a big number in case of non existing dates', () => {
+        // given
+        const date1 = undefined;
+        const date2 = undefined;
+
+        // when
+        const result = diffSeconds(date1, date2);
+
+        // then
+        expect(result).toEqual(9999999);
```

Comment:
this is only 115 days. Now that it's a general function we should probably increase it
maybe use MAX_SAFE_INTEGER

Reply:
Ah, good idea! Updated accordingly.